swarm/network/stream/delivery.go

```diff
@@ -244,6 +244,11 @@ R:
 			continue R
 		default:
 		}
+		if len(req.SData) > chunk.DefaultSize {
```

Comment:
Variable `chunk` here is no longer a package, but `*storage.Chunk`.

Reply:
Fixed